.lintstagedrc.json

```diff
@@ -1,3 +1,4 @@
 {
-  "!(docs/**/*)*.{ts,js,json,html,md,mts}": ["eslint --fix", "prettier --write"]
+  "!(docs/**/*)*.{ts,js,json,html,md,mts}": ["eslint --fix", "prettier --write"],
+  "cSpell.json": ["ts-node-esm bin/fixCSpell.ts"]
```

Comment:
Usually, I see people put this into a `scripts/` folder instead of a `bin/` folder.

I think `bin/` is normally only used if you're exporting a binary so that it can be used publicly.

Reply:
Moved